From 8da3bea72d22170ad0a2b2e199af0c364c098965 Mon Sep 17 00:00:00 2001
From: Feng Pei <quic_fpei@quicinc.com>
Date: Sat, 16 Dec 2023 18:43:11 +0800
Subject: [PATCH] hostapd: Add SCS OBJ for wpa_supplicant makefile

Add SCS UL Support for hostapd cause wpa_supplicant-macsec compilation
issue,there is no SCS object to link.

Signed-off-by: Feng Pei <quic_fpei@quicinc.com>
---
 wpa_supplicant/Makefile | 1 +
 1 file changed, 1 insertion(+)

--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -1046,6 +1046,7 @@ OBJS += ../src/ap/ieee802_11_he.o
 endif
 ifdef CONFIG_IEEE80211BE
 OBJS += ../src/ap/ieee802_11_eht.o
+OBJS += ../src/ap/scs.o
 endif
 ifdef CONFIG_WNM_AP
 CFLAGS += -DCONFIG_WNM_AP
